client/a8c-for-agencies/sections/client/client-landing.tsx

```diff
@@ -0,0 +1,86 @@
+import page from '@automattic/calypso-router';
+import { addQueryArgs, getQueryArg, getQueryArgs } from '@wordpress/url';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import Layout from 'calypso/a8c-for-agencies/components/layout';
+import LayoutBody from 'calypso/a8c-for-agencies/components/layout/body';
+import LayoutHeader, {
+	LayoutHeaderTitle as Title,
+} from 'calypso/a8c-for-agencies/components/layout/header';
+import LayoutTop from 'calypso/a8c-for-agencies/components/layout/top';
+import {
+	A4A_CLIENT_SUBSCRIPTIONS_LINK,
+	A4A_OVERVIEW_LINK,
+} from 'calypso/a8c-for-agencies/components/sidebar-menu/lib/constants';
+import { useSelector } from 'calypso/state';
+import { getActiveAgency, hasFetchedAgency } from 'calypso/state/a8c-for-agencies/agency/selectors';
+import { getClient, hasFetchedClient } from 'calypso/state/a8c-for-agencies/client/selectors';
+
+/**
+ * Redirect with Current Query
+ * Adds all of the current location's query parameters to the provided URL before redirecting.
+ */
+const redirectWithCurrentQuery = ( url: string ) => {
+	const args = getQueryArgs( window.location.href );
+	return page.redirect( addQueryArgs( url, args ) );
+};
+
+export default function ClientLanding() {
+	const translate = useTranslate();
+	const title = translate( 'Automattic for Agencies' );
+
+	const hasFetched = useSelector( hasFetchedClient );
+	const client = useSelector( getClient );
+
+	const hasFetchedAgencies = useSelector( hasFetchedAgency );
+	const agency = useSelector( getActiveAgency );
+
+	const redirectAgency = false; // FIXME: Remove this once we have the correct logic
```

Comment:
What should be the correct logic here?

Reply:
Correct logic is the authorization part that we will have. We just need to remove this because this is purely there for development.